thorlcr/graph/thgraphslave.cpp

```diff
@@ -128,6 +129,7 @@ CSlaveActivity::~CSlaveActivity()
 void CSlaveActivity::setInput(unsigned index, CActivityBase *inputActivity, unsigned inputOutIdx)
 {
     CActivityBase::setInput(index, inputActivity, inputOutIdx);
+    IHThorArg *helper;
```

Comment:
Why the new member variable for helper? It is already in the base class (baseHelper)